### ISSUE TITLE
migration: rethink the usage and content of the popovers in the dialog 

### ISSUE THREAD

**Issue by KKoukiou** [cockpit-project/cockpit-machines#164]
I find the content text no ideal, martin finds the popover itself not the right component here:

Martin: I don't like the popovers much either -- having some alert-like inline text in the dialog would feel nicer to me. But that can be a follow-up.

See: https://github.com/cockpit-project/cockpit-machines/pull/15#issuecomment-838755703

Worth doing one more iteration here.

**Comment by garrett** [comment]
I suggested some inline text, but was told in IRC that it should be tooltips instead. But PF says this type of content should not be tooltips, but popovers (and I agree; it's much too wordy for tooltips)... and so here we are.

For reference, here are mockups with a "notes" section (disregard the merging of the actions):

![migrate-vm - move, shared](https://user-images.githubusercontent.com/10246/117973717-042eac80-b32d-11eb-9a09-7c21456a21bd.png)

![migrate-vm - temporary](https://user-images.githubusercontent.com/10246/117973734-08f36080-b32d-11eb-878f-6f7b597f59ef.png)

So we simply cannot use tooltips for this content (as there's way too much), and enough of us don't like the inline text, and yet others don't like the popovers... then what?

**Comment by garrett** [comment]
Probably noteworthy; I also made a version with the notes outside of the form area (below it, basically):

![migrate-vm - temporary(1)](https://user-images.githubusercontent.com/10246/117973918-3dffb300-b32d-11eb-8f9f-9ced23cfeb9c.png)

**Comment by skobyda** [comment]
Now that some time passed, we could maybe come back and have a conclusion if it should be left as it is or needs some changes.

I also prefer Popovers over in-modal text.
Since, as @garrett said, they are not tooltips, but popovers. Popovers are meant to have both short or long text. They are meant to richer text (headlines, paragraphs... just like in our case), and they expand the information futher. So I think we are using popovers here correctly, for what they were meant to, and the text is not too long for a popover.

**Comment by kaechele** [comment]
Probably should have posted https://github.com/cockpit-project/cockpit-machines/issues/684#issuecomment-1631151786 into this issue instead.

**Comment by marusak** [comment]
Agree with Simon, we can close this
